pkg/services/ngalert/database_mig.go

```diff
@@ -68,3 +68,28 @@ func addAlertDefinitionVersionMigrations(mg *migrator.Migrator) {
 	mg.AddMigration("alter alert_definition_version table data column to mediumtext in mysql", migrator.NewRawSQLMigration("").
 		Mysql("ALTER TABLE alert_definition_version MODIFY data MEDIUMTEXT;"))
 }
+
+func alertInstanceMigration(mg *migrator.Migrator) {
+	alertInstance := migrator.Table{
+		Name: "alert_instance",
+		Columns: []*migrator.Column{
+			{Name: "definition_org_id", Type: migrator.DB_BigInt, Nullable: false},
+			{Name: "definition_uid", Type: migrator.DB_NVarchar, Length: 40, Nullable: false, Default: "0"},
+			{Name: "labels", Type: migrator.DB_Text, Nullable: false},
+			{Name: "labels_hash", Type: migrator.DB_NVarchar, Length: 190, Nullable: false},
+			{Name: "current_state", Type: migrator.DB_NVarchar, Length: 190, Nullable: false},
+			{Name: "current_state_since", Type: migrator.DB_BigInt, Nullable: false},
+			{Name: "last_eval_time", Type: migrator.DB_BigInt, Nullable: false},
+		},
+		PrimaryKeys: []string{"definition_org_id", "definition_uid", "labels_hash"},
+		Indices: []*migrator.Index{
+			{Cols: []string{"definition_org_id", "definition_uid", "current_state"}, Type: migrator.IndexType},
+			{Cols: []string{"definition_org_id", "current_state"}, Type: migrator.IndexType},
```

Comment:
@kylebrandt I've added this index since I think it's required by [listAlertInstances](https://github.com/grafana/grafana/blob/6f371b644b92ec6f00f12f78cf09c14545ab2cd6/pkg/services/ngalert/instance_database.go#L48) if the definition UID is missing by the command.